pkg/strategy/supertrend/strategy.go

```diff
@@ -413,6 +424,11 @@ func (s *Strategy) calculateQuantity(ctx context.Context, currentPrice fixedpoin
 	}
 }
 
+func (s *Strategy) CalcAssetValue(price fixedpoint.Value) fixedpoint.Value {
```

Comment:
You can use this method to calculate net asset `func (m BalanceMap) Assets(prices map[string]fixedpoint.Value, priceTime time.Time) AssetMap {`